src/System.Private.ServiceModel/src/System/ServiceModel/PeerHopCountAttribute.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.ServiceModel.Channels;
+
+namespace System.ServiceModel
+{
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = false)]
+    public sealed class PeerHopCountAttribute : MessageHeaderAttribute
+    {
+        public PeerHopCountAttribute()
+            : base()
+        {
+            base.Name = PeerStrings.HopCountElementName;
+            base.Namespace = PeerStrings.HopCountElementNamespace;
+            base.ProtectionLevel = ProtectionLevel.None;
+            base.MustUnderstand = false;
+        }
+        public new bool MustUnderstand
```

Comment:
These can use the new delegate syntactic sugar. e.g.:
```csharp
    public new bool MustUnderstand => base.MustUnderstand;
```
I've been using it lately and it really helps to make the code look cleaner, and the compiler emits identical instructions.